### PR TITLE
Update James' missions with new information

### PR DESCRIPTION
**Pull request by Pointedstick**

### CODE REVIEW

**Comment by Amazinite** [comment]
You know, there are quite a few things that James should probably mention that have been added more recently. How about we have one PR handled all that?

Here's the list of things I had written down that James should probably mention.

```
Talking about mechanics:

* Explain the raid system, especially to Star Barge starts
* Explain how being unarmed leads to pirates ignoring you more, especially to Star Barge and Shuttle starts
* Explain how checking the spaceport can lead to missions (instead of just implying it through having the James missions continuing through the spaceport)

Talking about gameplay styles:

* Explain mining to Sparrow starts

Talking about lore:

* Explain some of the game's in-universe history
* Explain about some of the regions of space
  * Pull this information from the timeline and player's manual on GitHub
  * Possibly use the codex idea from the "no need for outside info" card, having James tell the player about the log book before leaving and giving the player a data chip full of historical information.
```

That last category is less important for gameplay, and is more about helping the player better understand the story of the universe.

File: data/intro missions.txt [hunk 1]
```diff
@@ -382,7 +382,7 @@ mission "Intro [2 Interceptor]"
 				`	"Got it."`
 					accept
 				`	"I really shouldn't try to fight?"`
-			`	"It just isn't feasible," he says. "If a ship has turrets, your Sparrow can't get close enough to hit it without being damaged, and then it's just a matter of whose weapons and shields are stronger. And one heavy rocket hit could nearly disable you. So, don't risk it."`
+			`	"It just isn't feasible," he says. "If a ship has gun turrets, your Sparrow can't get close enough to hit it with your own fixed guns without being damaged. If it has anti-missile turrets, you can't count on hanging back and bombarding it with missiles from long distance - at least not with weapons can fit in a Sparrow. In this case it's just a matter of whose weapons and shields are stronger. And one heavy rocket hit could nearly disable you. So, don't risk it."`
```
Comment (Amazinite) [inline data/intro missions.txt]: `at least not with weapons can fit in a Sparrow.` -> ` at least not with weapons that can fit in a Sparrow.` ?

**Comment by Pointedstick** [comment]
Sounds good, I'll add those too.

**Comment by Amazinite** [comment]
Sitrep on this?

**Comment by Amazinite** [comment]
I'm working on this right now. Should have something tomorrow.

**Comment by Amazinite** [comment]
#4547 includes these changes and more, so I'm closing this PR in favor of that one. Would appreciate your feedback on it.

**Pull request closed without merging**